invokeai/app/api/routers/models.py

```diff
@@ -90,23 +49,28 @@ async def list_models(
     return models
 
 @models_router.post(
```

Comment:
This should probably be a `PATCH` if it's to change an existing entity

Reply:
Done.